### DISCUSSION TITLE
Show the open object on Database Navigator

### DISCUSSION THREAD

**Discussion by dxps** [dbeaver/dbeaver#34157] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
A nice feature would be to show the object on the database navigator, based on the open tab.

At the time of this writing - DBeaver version 24.0.5 - this feature is not available.

When you open DBeaver, the entries on the _Database Navigator_ are not expanded, so you may not know which already open (as a tab) object belongs to which connection and database.
![image](https://github.com/dbeaver/dbeaver/assets/14140226/12f2df44-0e9e-4900-b837-7e1ddea094dc)

This feature is common to IDEs like IntellliJ or Visual Studio Code, like this:
![image](https://github.com/dbeaver/dbeaver/assets/14140226/bc0c9524-b9a7-4465-a1bc-cfb4783cd80c)

**Describe the solution you'd like**
Be able to have the option on the contextual menu (right-click on that open object tab) to identify the object on the _Database Navigator_.

Thanks in advance for considering it! :pray:

**Comment by E1izabeth** [comment, marked as answer]
Hi, @dxps.
Have you tried using `Link with editor` ![Link-with-Editor-icon](https://github.com/dbeaver/dbeaver/assets/28875055/e04ee944-1d99-498e-989c-c18c2e9a21aa) button?
 
![image](https://github.com/dbeaver/dbeaver/assets/28875055/7512c2f9-ce70-4f94-a915-505037baddd4)

**Comment by dxps** [comment]
Hi Elisabeth,

Thanks for the hint! :pray:  Indeed, that option is handy as well. I must have missed that.
I'll close the ticket as I consider having enough for the stated need.

All the best and keep up the good work!
